### ISSUE TITLE
Missing requirements.txt for Dependency Management

### ISSUE THREAD

**Issue by dfuchss** [aiopsplus/C4RLLaMA#1]
I noticed that the repository does not contain a requirements.txt file, which makes it difficult to determine the exact package versions needed to run the code successfully. Attempting to use the current versions of the packages does not seem to work as expected.

I was able to find a configuration that works, but I am uncertain whether it is the correct one. Also the version of python would be good to know.

Would it be possible to add a requirements.txt file or provide a list of the specific package versions required to ensure proper functionality of the code?

Thank you for your assistance!

**Comment by dfuchss** [comment]
Ok, I'm quite sure that I don't have the right versions;

I'm getting a lot of warnings like "The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation."

**Comment by KubraCaglar** [comment]
I'm having the same issues, too. Did you find the right versions?

**Comment by dfuchss** [comment]
Unfortunately, I also have no further information. I guess the authors did not participate in the artifact evaluation. So probably the only thing you can do is writing an email to them .

**Comment by inkeast** [comment]
Thank you for your attention to our work. The [requirements.txt](https://github.com/aiopsplus/C4RLLaMA/blob/main/requirements.txt) has been updated.
